go.mod

```diff
@@ -3,13 +3,18 @@ module github.com/ava-labs/simplex
 go 1.23.0
 
 require (
-	github.com/stretchr/testify v1.9.0
+	github.com/StephenButtolph/canoto v0.17.2
```

Comment:
We don't need canoto to serialize two uint64, especially not for empty votes which we know should contain as less information as possible.